src/main/webapp/app/shared/components/code-button/code-button.component.ts

```diff
@@ -238,6 +245,7 @@ export class CodeButtonComponent implements OnInit {
      * Add the credentials to the http url, if a token should be used.
      *
      * @param insertPlaceholder if true, instead of the actual token, '**********' is used (e.g. to prevent leaking the token during a screen-share)
+     * @param alwaysHttpWithToken if true, the http url is always used, even if the ssh url is configured (used for Theia)
```

Comment:
I cannot see this param `alwaysHttpWithToken` in the method, please fix the JSDoc